build.gradle

```diff
@@ -11,19 +11,28 @@ version '1.0-SNAPSHOT'
 sourceCompatibility =  JavaVersion.VERSION_11  // source-code version and must be <= targetCompatibility
 targetCompatibility = JavaVersion.VERSION_11 // bytecode target version
 
-repositories {
-    jcenter()
+allprojects {
+    repositories {
+        jcenter()
+        maven { url "https://jitpack.io" }
+    }
 }
 
 dependencies {
+    implementation group: 'com.github.BIBSYSDEV', name: 'nva-commons', version: 'v0.3.2'
     implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.9'
     implementation group: 'javax.ws.rs', name: 'javax.ws.rs-api', version: '2.1.1'
     implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
     implementation group: 'org.slf4j', name: 'slf4j-api', version: '1.7.29'
     implementation group: 'org.jsoup', name: 'jsoup', version: '1.12.1'
     implementation group: 'com.amazonaws', name: 'aws-lambda-java-core', version: '1.2.0'
     implementation group: 'com.amazonaws', name: 'aws-java-sdk-s3', version: '1.11.715'
+    implementation group: 'org.glassfish.jersey.core', name: 'jersey-common', version: '2.22.2'
+    implementation group: 'org.zalando', name: 'problem', version: '0.23.0'
+    implementation group: 'org.zalando', name: 'jackson-datatype-problem', version: '0.23.0'
 
+    testImplementation group: 'org.glassfish.jersey.core', name: 'jersey-common', version: '2.22.2'
```

Comment:
why do you need to repeat the dependency?

Reply:
I was using `MediaType.valueOf(input.getMimetype());
` and it gave me a class not found exception without that dependency.